Signal/src/view controllers/CallViewController.swift

```diff
@@ -244,8 +244,12 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
         declineIncomingButton = createButton(imageName:"hangup-active-wide",
                                              action:#selector(didPressDeclineCall))
 
+        let spacerView = UIView(frame: CGRect(x: 0, y: 0, width: buttonSize(), height: 0))
+        spacerView.autoSetDimension(.width, toSize:buttonSize())
+        spacerView.autoSetDimension(.height, toSize:buttonSize())
+
         incomingCallView = createContainerForCallControls(controlGroups : [
-            [acceptIncomingButton, declineIncomingButton ]
+            [acceptIncomingButton, spacerView, declineIncomingButton ]
```

Comment:
I'd like to suggest that they are too far apart in the screenshots above. They should both be easily accessible with the thumb, even on large form factors like the 7+.